README.md

```diff
@@ -30,14 +30,28 @@ To allow for more complex configurations, you can also add a top-level section `
             "version": "1.6.1"
             "filePattern": "electron$"
         },
+        "system_uri": {
```

Comment:
I would keep the examples clean and only refer to the electron package, so perhaps simply show how to disable electron download for `mobile` env var should be good enough.